stubs/pyjks/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,22 @@
+jks.PrivateKeyEntry.algorithm_oid
+jks.PrivateKeyEntry.pkey
+jks.PrivateKeyEntry.pkey_pkcs8
+jks.SecretKeyEntry.algorithm
+jks.SecretKeyEntry.key
+jks.SecretKeyEntry.key_size
+jks.__all__
+# Attributes implemented by __getattr__
+jks.jks.PrivateKeyEntry.algorithm_oid
+jks.jks.PrivateKeyEntry.pkey
+jks.jks.PrivateKeyEntry.pkey_pkcs8
+jks.jks.SecretKeyEntry.algorithm
+jks.jks.SecretKeyEntry.key
+jks.jks.SecretKeyEntry.key_size
+jks.PrivateKeyEntry.algorithm_oid
```

Comment:
isn't this the same as the allowlist entry on line 1?

Reply:
Ah yes, my bad, I got confused because my git push failed.

Reply:
oops, sorry 😶